src/applepay.js

```diff
@@ -37,12 +38,14 @@ async function createOrder(
   payload: OrderPayload
 ): Promise<CreateOrderResponse> {
   const basicAuth = btoa(`${getClientID()}`);
+  const partnerAttributionId = getPartnerAttributionID();
 
```

Comment:
We dont support Client Side Create Order right. I don't think we ask Merchants to use this at all. This method to `CreateOrder ` can actually be removed from the Component . We can add to just `ApproveApplePayment` call

Reply:
I wasn't sure if this was being used as a placeholder for future use. If it were, we'd want to get the partnerAttributionId in this call.

Reply:
Yeah, To give some background context. This was added as a plan to Support Client Side Create Order But it is being decided that we will no longer support Client Side Order Creation not just for ApplePay but across the board. 

A Follow up PR would suffice